style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  box-size: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+@import url(https://fonts.googleapis.com/css?family=roboto);
+
+html {
+  font-family: "roboto", arial;
+  font-size: 18px;
+}
+
+nav {
+  justify-content: center;
+  padding: 20px;
+  background-color: #046c94;
+  color: #fff;
+  list-style-type: none;
+}
+
+.nav-links {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-links li {
+  list-style-type: none;
+}
+
+a {
+  color: #fff;
+  text-decoration: underline;
+}
+
+body, html {
+  height: 100%;
+  margin: 0;
+  background-color: #000;
+  color: #fff;
+}
+
+.hero-image {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+    url("https://images.unsplash.com/photo-1555099962-4199c345e5dd?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=1350&q=80");
+  height: 50%;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.hero-text {
+  text-align: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: #fff;
+  font-size: 25px;
+}
+
+.hero-text button {
+  border: none;
+  outline: 0;
+  display: inline-block;
+  padding: 10px 25px;
+  color: #fff;
+  background-color: #046c94;
+  text-align: center;
+  cursor: pointer;
+}
+
+.hero-text button:hover {
+  background-color: #34c2b2;
+  color: #fff;
+}
+
+section {
+  width: 800px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+header {
+  padding: 50px 0;
+}
+
+h2 {
+  margin: 0;
+  font-size: 40px;
+  margin-top: 30px;
+}
+
+section img {
+  height: auto;
+  width: 100%;
+}
+
+section img:hover {
+  transform: scaleY(1.3);
+  -webkit-filter: grayscale(50%);
+  filter: grayscale(50%);
+}
+
+#projects {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+}
+
+.fa {
+  padding: 20px;
+  font-size: 30px;
+  width: 50px;
+  text-decoration: none;
+  margin-top: 30px;
+}
+
+.fa:hover {
+  opacity: 0.7;
+}
+
+.fa-envelope {
+  background: #007bb5;
+  color: #fff;
+}
+
+.fa-github {
+  background: #55acee;
+  color: #fff;
+}
+
+.fa-twitter {
+  background: #55acee;
+  color: #fff;
+}
+
+.fa-linkedin {
+  background: #007bb5;
+  color: #fff;
+}
```

Comment:
You're setting all of these icons so they're white (which is fine!), but rather than setting them individually you could probably just move the color onto your .fa selector, which applies to all of the icons.